File: node/search-by-queryget.js
const util = require('util');
const request = require("request-promise");

let esconn = require("../esconn.json");

// search by name with querystring
request.get({
    uri                    : esconn.uri+'/node/profile/_search?q=step:3&pretty',
    resolveWithFullResponse: true,
    json: true
}).then((response) => {
    console.log('search by name with querystring');
    console.log(`elasticsearch response : ${util.inspect(response.body, false, null)}`);
    return true
});

File: python/search-by-queryget.py
import json
import requests

esconn = json.load(open('../esconn.json'))


r = requests.get("%s%s" % (esconn['uri'],'/node/profile/_search?q=step:3&pretty'))
print(r.status_code, r.reason)
parsed = json.loads(r.content)
print (json.dumps(parsed, indent=4, sort_keys=True))







